lstchain/scripts/lstchain_mc_dl1_to_dl2.py

```diff
@@ -62,6 +62,15 @@ def main():
     config = replace_config(standard_config, custom_config)
 
     data = pd.read_hdf(args.datafile, key=dl1_params_lstcam_key)
+
+    # Dealing with pointing errors
+    if 'alt_tel' in data.columns and 'az_tel' in data.columns:
+        # make sure there is a least one good pointing value to interp from.
+        if np.isfinite(data.alt_tel).any() and np.isfinite(data.az_tel).any():
+            impute_pointing(data)
+        else:
+            data.alt_tel = 0
```

Comment:
what would happen if instead of 0 we put these as negative values?
I'm a bit worried about the 0s in case of errors, because in principle we could observe (alt,az) = (0,0), specially if we forget. If the negative values do not work, could you add a more extensive comment about this?

Reply:
That would be pointing at *very* low zenith but I imagine we could in principle.
`(alt, az)=(-90,-90)deg` should work as well.